client/js/actions/view-action-creator.js

```diff
@@ -1,9 +1,15 @@
-var Dispatcher = require('./dispatcher/dispatcher');
+var AppDispatcher = require('../dispatcher/dispatcher');
 var ServerActionCreator = require('./server-action-creator');
-var AppConstants = require('./constants/constants');
+var ApiUtils = require('../utils/api-utils');
+var AppConstants = require('../constants/constants');
 
 var ViewActionCreator = {
-
+  addEvent: function (event) {
+    AppDispatcher.dispatch({
+      actionType: AppConstants.ViewActionTypes.ADD_EVENT
+    });
+    ApiUtils.addEvent(event);
```

Comment:
i think we sould pass ServerActionCreator.eventAdded as the callback to APIutils.addEvent. thoughts?